packages/mobile_app/lib/screens/registration/screen.dart

```diff
@@ -165,13 +166,15 @@ class RegistrationScreen extends StatelessWidget {
                     month.wrapped.isEmpty ||
                     day.wrapped.isEmpty ||
                     year.wrapped.isEmpty) {
-                  _alertWrongInformation(context,
-                      title: lang.notAllFieldsFilled);
+                  showAlert(
```

Comment:
Just curious, why are we showing alert when doing validation? I thought we agreed on not doing it.

Reply:
It is done to place some widget with transperent screen over current screen with z value, no matter what z value was before it.

You said to inform user if he clicks button while not accepting terms or not given information ot his birth date.

Reply:
Ohhh, @rimmer it is padding for our gradient button in registration screen, it must be fixed